ethereum/spec/src/main/java/tech/pegasys/teku/spec/Spec.java

```diff
@@ -197,11 +196,24 @@ public SpecConfig getGenesisSpecConfig() {
     return getGenesisSpec().getConfig();
   }
 
+  /**
+   * Base networking constants
+   *
+   * <p>These constants are unified among forks and are not overriden, new constant name is used if
+   * it's changed in the new fork
+   */
   public NetworkingSpecConfig getNetworkingConfig() {
     // Networking config is constant along forks
     return getGenesisSpec().getConfig().getNetworkingConfig();
   }
 
+  /** Networking config with Deneb constants */
+  public Optional<NetworkingSpecConfigDeneb> getNetworkingConfigDeneb() {
```

Comment:
Even if this is looks bad, we have to deal with it. As mentioned in the previous method javadoc, there is no override. Fork specific code will refer to fork specific configs. At the moment this still holds, but maybe in the future we could have some ugly things where the same logic will need to switch config (it will be a de-facto config param override). ATM I wouldn't do anything different than this.